python/cuml/test/dask/test_label_encoder.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import cuml
+from cuml.dask.preprocessing.LabelEncoder import LabelEncoder
+import cudf
+import numpy as np
+import dask_cudf
+import pytest
+from cuml.common.exceptions import NotFittedError
+import cupy as cp
+
+
+def _df_to_similarity_mat(arr):
+    arr = arr.reshape(1, -1)
```

Comment:
I'm not sure I understand this function... it seems to take an array rather than a df?